metricbeat/module/elasticsearch/elasticsearch.go

```diff
@@ -167,3 +170,39 @@ func GetNodeInfo(http *helper.HTTP, uri string, nodeID string) (*NodeInfo, error
 	}
 	return nil, fmt.Errorf("no node matched id %s", nodeID)
 }
+
+// GetLicense returns license information
+func GetLicense(http *helper.HTTP, resetURI string) (map[string]interface{}, error) {
```

Comment:
Should we cache this value for a few minute as the license is hopefully very rarely going to change?

Reply:
Good idea. Will do.

Reply:
Implemented in e72f9c90c. Curious to hear your thoughts on the implementation — too complex, too simple, just right for now?